casepro/settings_common.py

```diff
@@ -56,6 +56,17 @@
 SITE_BACKEND = 'casepro.backend.NoopBackend'
 SITE_ANON_CONTACTS = False
 
+# junebug configuration
```

Comment:
This is fine for now but if we find ourselves needing lots of backend specific settings maybe we could move such things into `settings_rapidpro.py.dev` and `settings_junebug.py.dev`